support/client/lib/vwf/view/mil-sym.js

```diff
@@ -379,7 +379,15 @@ define( [ "module", "vwf/view", "mil-sym/cws", "jquery" ], function( module, vie
         // Set affiliation in symbol id
         symbolCode = cws.addAffiliationToSymbolId( symbolID, affiliation );
         
-        var img = rendererMP.RenderSymbol2D("ID","Name","Description", symbolCode, milSymControlPts, bounds[0], bounds[1], null, modifiers, format);
+        // Convert modifiers to actual names and types
+        var actualNameModifiers = {};
+        for ( mod in modifiers ) {
+            var modObj = cws.modifierByAlias( mod );
+            var modActualName =  this.state.getModifierActualName( modObj );
+            actualNameModifiers[ modActualName ] = this.state.convertModifierValue( modObj, modifiers[ mod ] );
```

Comment:
This is fine.  I feel a little more comfortable using `self` in these contexts because I'm never quite sure what `this` is going to be.  But in this case, it should always be the right `this`.  So, up to you whether you want to switch out `this` for `self`.  :smile: 

I would also like us to rename `self` to `viewDriver` at some point so it has some meaning, rather than just being a generic variable name like `thing`, but that, of course, is outside the scope of your PR.